design_pim/overview.rst

```diff
@@ -0,0 +1,354 @@
+
+Register it
++++++++++++
+
+Now that we created our form extension we need to register it:
+
+.. code-block:: yaml
+
+    # src/Acme/Bundle/CustomBundle/Resources/config/form_extensions.yml
+    extensions:
+        my_form_extension:
+            module: my_custom_view             # Your requirejs module name
+            parent: pim-product-edit-form-meta # The parent of your extension (the meta section of the product edit form in this case)
+
+
+A fiew words about this small configuration:
+
+- the key ``my_form_extension`` should be a unique key to represent your form extension.
+- you can declare multiple form extensions with the same requirejs module.
+- to be registered your configuration file should be named ``form_extensions.yml`` or putted in a ``form_extensions`` folder in your bundle ``Resources/config`` folder.
+- You can override any form extension by using the same unique key (the order of the override is defined by your ``AppKernel.php`` registration order).
+
+This configuration is the minimal example. Here are the other parameters that you can use:
+
+.. code-block:: yaml
+
+    # src/Acme/Bundle/CustomBundle/Resources/config/form_extensions.yml
+    extensions:
+        my_form_extension:
+            module: my_custom_view
+            parent: pim-product-edit-form-meta
+            targetZone: header       # Where to put the extension in the parent DOM
+            position: 90             # Its position among the other extensions
+            aclResourceId: pim_catalog_product_edit # The acl required to display this extension. Is the user doesn't have the right to access this ACL, the extension will not be registered
+            config:
+                here: you_can_put
+                whatever: you_want
+
+
+The last key of our module (``config``) is used to pass the configuration you want to the requirejs module. You can get it in the initialize method of your module:
+
+.. code-block:: javascript
+    :linenos:
+
+    /**
+     * {@inheritdoc}
+     */
+    initialize: function (meta) {
+        this.meta = meta.config;
+
+        BaseForm.prototype.initialize.apply(this, arguments);
+    }
+
+
+As you can see, we receive the entire configuration through the constructor of our view.
+
+Now that our extension is registered, we need to clear the symfony cache and we are good to go. You should see your extension in the meta section of the product edit form now.
+
+Some extra features
+===================
+
+Useful methods
+++++++++++++++
+
+Here are a list of method that you can override or call in your extension that should make your life easier.
+
+Managing the model
+******************
+
+Each form have an internal model representing the current object we are modifying. Here is the way to access it and modify it
```

Comment:
You should say that `getFormData()` doesn't return the model of the view itself but the root's one.

Reply:
DOne :)